Specs/Scene/ModelExperimental/I3dmLoaderSpec.js

```diff
@@ -267,22 +268,13 @@ describe("Scene/ModelExperimental/I3dmLoader", function () {
       );
 
       const transform = loader.components.transform;
-      expect(transform[Matrix4.COLUMN0ROW0]).toEqual(1.0);
-      expect(transform[Matrix4.COLUMN1ROW0]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN2ROW0]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN3ROW0]).toEqual(1215013.8340490046);
-      expect(transform[Matrix4.COLUMN0ROW1]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN1ROW1]).toEqual(1.0);
-      expect(transform[Matrix4.COLUMN2ROW1]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN3ROW1]).toEqual(-4736316.75897742);
-      expect(transform[Matrix4.COLUMN0ROW2]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN1ROW2]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN2ROW2]).toEqual(1.0);
-      expect(transform[Matrix4.COLUMN3ROW2]).toEqual(4081608.4380407534);
-      expect(transform[Matrix4.COLUMN0ROW3]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN1ROW3]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN2ROW3]).toEqual(0.0);
-      expect(transform[Matrix4.COLUMN3ROW3]).toEqual(1.0);
+      // prettier-ignore
+      expect(transform).toEqualEpsilon(new Matrix4(
```

Comment:
It seems like the epsilon is missing from this function -- maybe add one or change this to `toEqual`